lib/matplotlib/axes/_base.py

```diff
@@ -3902,18 +3902,18 @@ def _make_twin_axes(self, *kl, **kwargs):
 
     def twinx(self):
         """
-        Create a twin Axes sharing the xaxis
+        Create a twin Axes instance sharing the x-axis with self
 
-        Create a twin of Axes for generating a plot with a shared
-        x-axis but independent y-axis.  The y-axis of self will have
-        ticks on left and the returned axes will have ticks on the
-        right. To ensure tick marks of both axis align, see
-        `~matplotlib.ticker.LinearLocator`
+        Create a twin Axes instance for generating a plot with a shared
```

Comment:
also not sure this is any clearer. docs support images, so maybe that's the way to go here? 
https://github.com/matplotlib/matplotlib/blob/master/lib/matplotlib/axes/_axes.py#L1660